김현수/mvvmstudy/app/src/main/java/com/example/mvvmstudy/Adapter/BindingAdapter.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.mvvmstudy.Adapter
+
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.databinding.BindingAdapter
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.bumptech.glide.request.RequestOptions
+import com.example.mvvmstudy.R
+
+@BindingAdapter("imageUrl")
+fun loadImage(
+    imageView: ImageView,
+    url: String
+) {
+    Glide.with(imageView.context)
+        .load(url)
+        .placeholder(ContextCompat.getDrawable(imageView.context, R.drawable.ic_baseline_storage))
+        .error(ContextCompat.getDrawable(imageView.context, R.drawable.ic_baseline_storage))
```

Comment:
placeHolder와 error도 매개변수로 주입받아 더욱 일반화해서 쓸 수 있을 것 같아요!
```suggestion
@BindingAdapter(value=["app:imageUrl", "error"], requireAll = false)
fun loadImage(
    imageView: ImageView, 
    url: String,
    @DrawableRes placeHolderRes: Int = R.drawable.ic_baseline_storage
    @DrawableRes errorDrawableRes: Int = R.drawable.ic_baseline_storage
) {
    Glide.with(imageView.context)
        .load(url)
        .placeholder(ContextCompat.getDrawable(imageView.context, placeHolderRes))
        .error(ContextCompat.getDrawable(imageView.context, errorDrawableRes))
```